cairo-sys-rs/build.rs

```diff
@@ -4,7 +4,8 @@ use pkg_config::{Config, Error};
 use std::env;
 use std::io;
 use std::io::prelude::*;
-use std::process;
+use std::path::{Path, PathBuf};
```

Comment:
In here: two possibilities:

 1. you move the import into the function so no need to put conditional compilation on them
 2. you put conditional compilation on them

:)